src/jit/lsra.cpp

```diff
@@ -12205,6 +12371,10 @@ void LinearScan::verifyFinalAllocation()
             VARSET_ITER_INIT(compiler, iter, currentBlock->bbLiveIn, varIndex);
             while (iter.NextElem(&varIndex))
             {
+                if (localVarIntervals[varIndex] == nullptr)
+                {
+                    continue;
+                }
                 regNumber regNum                  = getVarReg(inVarToRegMap, varIndex);
```

Comment:
No - because if it is not a candidate, that method will assert. I think I had missed a couple of places which I've now updated, but this one (and the others like it) can't use that. I've added an assert, though.